gatsby-node.ts

```diff
@@ -24,6 +24,7 @@ try {
 
 // ideally problems would be its own query with
 // source nodes: https://www.gatsbyjs.com/docs/reference/config-files/gatsby-node/#sourceNodes
+if (fs.existsSync('ids.log')) fs.rmSync('ids.log');
```

Comment:
does this just delete ids.log?

Reply:
yeah if it gets deleted if it already exists so basically it clears the log so it doesn't grow too big locally